### PR TITLE
draft sustainability episode

### PR DESCRIPTION
**Pull request by wikfeldt**

### CODE REVIEW

File: _episodes/05-partnership.md [hunk 1]
```diff
@@ -13,10 +13,51 @@ keypoints:
...
+until autumn 2021. A key objective for CodeRefinery 2.0 is to **render the project largely self-sustained after 
+October 2021**. 
 
 
+> ## Public documents 
+> 
+> Documents relating to the management of the CodeRefinery project, 
+> including project directive and project plan, are available on 
+> [this NeIC wiki page](https://wiki.neic.no/wiki/CodeRefinery)
+{: .callout}
```
Comment (bast) [inline _episodes/05-partnership.md]: They in fact are also available on neic.no. Sadly it's duplicated which for me means that I always forget to update one of the two places. Anyway, I will update the link after I add missing link to neic.no.

**Comment by rkdarst** [comment]
Would this be a good place to mention our/my grand plan of connecting CodeRefinery (teaching), RSEs (hands-on support), and NordicHPC (usable infra) in a complete nordic scientific computing package (and ask how to make that possible)?

**Comment by wikfeldt** [comment]
that would fit nicely!

**Comment by bast** [comment]
Thanks for the work so far. Looks great!

**Comment by wikfeldt** [comment]
thanks. i'll mark this ready for review now and we can follow up in future PRs

**Comment by bast** [comment]
I moved Richard's comment to the issue thread. Merging.

**Pull request merged**